### ISSUE TITLE
Comments to documentation

### ISSUE THREAD

**Issue by jfaraudo** [sntioudis/papreca#19]
Comments to the documentation

**Comment by jfaraudo** [comment]
In general, the documentation is clear and covers all relevant aspects. I have only two minor comments:
- For me, the flowchart in the section "Essential information (See before you start)"  athttps://sntioudis.github.io/papreca/md_essentials.html  is a bit confusing. I agree that all the information is correct. But at a first look , for an expert in MD like me, it looks pretty strange to see, in the MD part,  the sequence of minimization - MD run - and minimization again, because it is pretty unusual in the MD community. I understood that flow when looking at your paper  Computational Materials Science, vol. 229, 2023. The flow in Figure 1 of that paper is much clear to me. It is essentially the same, but it contains a schematic with a PES and the exploration over the PES implied by these steps. This makes it more clear to me. Also, the general organization of the flowchart is better. I suggest considering an update of this flowchart recycling elements of (or based on) that previous figure. 
- I do not understand the reason of the comment about LJ units in the section "PAPRECA units". I agree that there is no simple way of converting LJ units to real time, that's clear. But imagine that we select LJ units in LAMMPS. Then, if PAPRECA is also working in LJ units (if I understand correctly, PAPRECA uses the same units as LAMMPS), why not providing the time of MC, MD and total time in LJ units (no need of conversion to seconds)?

**Comment by jfaraudo** [comment]
All of these sounds good to me. I do not think that the "LJ community" so to speak is really working a lot on kMC these days. So I think it is not a big priority to implement a LJ units converter. But I think it is worth explaining in more detail that PAPRECA uses "real" units so all other LAMMPS units options are handled by internal conversion to "real" at PAPRECA.

**Comment by sntioudis** [comment]
1) I updated the general PAPRECA flowchart in the [Essentials section](https://sntioudis.github.io/papreca/md_essentials.html#flowchart). I realized that the General flowchart in the documentation with the two minimization steps (i.e., one before and one after executing the MD trajectory) was referring to the phosphates examples (and to our [previous paper]). PAPRECA is a more general software, in the sense that the user has complete freedom over the MD simulation options. For example, someone might choose to perform only one minimization (i.e., before the MD trajectory), or none (as exhibited in some of the examples). Therefore, I removed the specific steps and the flowchart now depicts a general MD stage (without further description).


2) A more detailed discussion regarding the units of PAPRECA is now in the [essentials](https://sntioudis.github.io/papreca/md_essentials.html#units) section.
